tests/testthat/test-SummarizeCrossStudy.R

```diff
@@ -0,0 +1,248 @@
+# Helper function to create test data
+create_test_results <- function() {
+  # Create test results with Analysis_srs0001 (site risk scores)
+  tibble::tibble(
+    GroupID = rep(c("Site001", "Site002", "Site003"), each = 2),
+    GroupLevel = "Site",
+    StudyID = rep(c("Study1", "Study2"), 3),
+    MetricID = "Analysis_srs0001",
+    Score = c(1.5, 2.3, 0.8, 1.2, 3.1, 2.9),
+    Numerator = c(10, 15, 5, 8, 20, 18),
+    Denominator = c(100, 120, 80, 90, 150, 140),
+    Metric = c(0.1, 0.125, 0.0625, 0.089, 0.133, 0.129),
+    Flag = c(1, 2, 0, 1, 2, 2),
+    SnapshotDate = as.Date("2025-01-01")
+  )
+}
+
+create_test_groups <- function() {
+  # Create test groups data with investigator names
+  tibble::tibble(
+    GroupID = rep(c("Site001", "Site002", "Site003"), each = 2),
+    GroupLevel = "Site",
+    StudyID = rep(c("Study1", "Study2"), 3),
+    Param = "InvestigatorLastName",
+    Value = c("Smith", "Smith", "Jones", "Jones", "Brown", "Williams")
+  )
+}
+
+test_that("Returns correct structure with required columns - #71", {
```

Comment:
```suggestion
test_that("Returns correct structure with required columns - (Issue #71)", {
```

Do we want to have any standardized formatting this would match the one that fixes #107

Reply:
I *think* the parser is fairly flexible, but will confirm during pilot.